crates/cubecl-runtime/src/tune/tuner.rs

```diff
@@ -1,16 +1,16 @@
-#[cfg(target_family = "wasm")]
-use web_time::Duration;
+use async_channel::{Receiver, Sender};
 
-#[cfg(not(target_family = "wasm"))]
-use core::time::Duration;
 use core::{any::Any, mem::ManuallyDrop};
-#[cfg(feature = "std")]
-use std::panic::{catch_unwind, resume_unwind, AssertUnwindSafe};
+use web_time::Duration;
```

Comment:
Do we always want to use duration from `web_time` ? Maybe we should put those things in `cubecl_common` ?

Reply:
Ah yes - web_time just compiles to normal time on non-wasm platforms and strips away all it's code, so besides a tiny compilation overhead there's no reason not to, so don't think moving it helps much either, but lmk!